## 通用参数
float64 maxSpeed
## 跟随参数
int8 useLoaclSlow
float64 endPathDis
float64 pathSlowDisThre
float64 getPath_speed
float64 path_zero_bias
float64 goalSlowDisThre
float64 getGoal_speed
float64 goal_zero_bias
float64 spinSpeed
int8 useCloudSlowDown
float64 minSpeed
float64 curvature
float64 slowBegin
int8 safetyStop
float64 maxAddAccel
float64 maxSlowAccel
float64 yawRateGain
float64 stopYawRateGain
float64 maxYawRate
float64 maxStopYawRate
int8 goal_path_direct
int8 use_MIDPlanning_slow
float64 MIDPlanning_slow_rate
float64 MIDPlanning_minSpeed
float64 dirDiffThre
int8 use_closeGoal_direct
float64 closeGoal_direct_dis
float64 quick_turn_N
